filecoin-proofs/src/bin/paramcache.rs

```diff
@@ -40,6 +44,16 @@ fn cache_post_params(post_config: PoStConfig) {
     info!(FCP_LOG, "begin PoSt parameter-cache check/populate routine for {}-byte sectors", n; "target" => "paramcache");
 
     let post_public_params = post_public_params(post_config);
+
+    {
+        let post_circuit: VDFPoStCircuit<Bls12> =
+            <VDFPostCompound as CompoundProof<
+                Bls12,
+                VDFPoSt<PedersenHasher, Sloth>,
+                VDFPoStCircuit<Bls12>,
+            >>::blank_circuit(&post_public_params, &ENGINE_PARAMS);
+        let _ = VDFPostCompound::get_param_metadata(post_circuit, &post_public_params);
```

Comment:
What is the effect of this code block?

Reply:
It generates the Groth parameter metadata file for VDF PoSt, which includes (for instance) the sector size.